cfgov/jinja2/v1/_includes/organisms/full-width-text.html

```diff
@@ -29,12 +29,14 @@
                 {% set media = image(block.value, 'original') %}
                 <img class="m-full-width-media_image" src="{{ media.url }}" alt="{{ media.alt }}">
             </div>
-        {% elif 'image_inset' in block.block_type %}
-            {{ block | safe }}
-        {% else %}
+        {% elif block.block_type in ['quote', 'cta', 'related-links'] %}
             <div class="m-inset">
                 {{ block | safe }}
             </div>
+        {% elif 'reusable_text' == block.block_type %}
```

Comment:
Yes, this logic makes me sad, but I also don't know a simpler better way right now. The work in GHE#378 might enable an alternate approach, but at root it does seem like the issue is that snippets don't know how to render themselves in the same ways that other blocks do. At least with other blocks you can define a template that gets used when the block is rendered.

One possible approach might be to derive a custom class like `ReusableTextChooserBlock` from [`SnippetChooserBlock`](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailsnippets/blocks.py#L9), that has a `class Meta: template = ...`.  That way you could encapsulate the logic of "how to render ReusableText snippets" in the class template itself.